### PR TITLE
Use apidoc directive in typed/fault-tolerance.md (#22904)

### PR DESCRIPTION
**Pull request by Captain1653**
References #22904

### CODE REVIEW

File: akka-docs/src/main/paradox/typed/fault-tolerance.md [hunk 1]
```diff
@@ -116,9 +116,9 @@ Java
...
+If child actors are created from @apidoc[setup](typed.*.Behaviors$) {scala="#setup[T](factory:akka.actor.typed.scaladsl.ActorContext[T]=%3Eakka.actor.typed.Behavior[T]):akka.actor.typed.Behavior[T]" java="#setup(akka.japi.function.Function)"} like in the previous example and they should remain intact (not stopped)
+when parent is restarted the @apidoc[supervise](typed.*.Behaviors$) {scala="#supervise[T](wrapped:akka.actor.typed.Behavior[T]):akka.actor.typed.scaladsl.Behaviors.Supervise[T]" java="#supervise(akka.actor.typed.Behavior)"} should be placed inside the @apidoc[setup](typed.*.Behaviors$) {scala="#setup[T](factory:akka.actor.typed.scaladsl.ActorContext[T]=%3Eakka.actor.typed.Behavior[T]):akka.actor.typed.Behavior[T]" java="#setup(akka.japi.function.Function)"} and using
+@scala[@scaladoc[SupervisorStrategy.restart.withStopChildren(false)](akka.actor.typed.RestartSupervisorStrategy#withStopChildren(enabled:Boolean):akka.actor.typed.RestartSupervisorStrategy)]@java[@javadoc[SupervisorStrategy.restart().withStopChildren(false)](akka.actor.typed.RestartSupervisorStrategy#withStopChildren(boolean))]
```
Comment (Captain1653) [inline akka-docs/src/main/paradox/typed/fault-tolerance.md:121]: I don't like the long line `SupervisorStrategy.restart.withStopChildren(false)`, but I need some different texts:
scala - `SupervisorStrategy.restart.withStopChildren(false)` (`restart` and then POINT)
java - `SupervisorStrategy.restart.withStopChildren(false)` (`restart` and then PARENTHESES).

I can use apidoc and the same text. Should I do it?

Comment (raboof) [reply akka-docs/src/main/paradox/typed/fault-tolerance.md:121]: I don't have a preference.

**Review by raboof — APPROVED** [review]

**Review by johanandren — APPROVED** [review]
LGTM

Long line doesn't matter to me either, so let's merge this.

**Comment by Captain1653** [comment]
@johanandren Thank you for your review. Is it worth to add the milestone for similar PR #30609  (if it is still possible for closed PR)

**Pull request merged**